webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/Internationalization/MessagesBundle.properties

```diff
@@ -0,0 +1,23 @@
+Scramble = Scramble
```

Comment:
Why does this exist in addition to MessagesBundle_en_US.properties?

Reply:
Removed. I just assumed this was some standard proceeding in i18n (https://docs.oracle.com/javase/tutorial/i18n/intro/after.html). It's working fine without the file.